index.html

```diff
@@ -419,7 +532,16 @@ <h3>
           information against unauthorized access. If location information is
           stored, users need to be allowed to update and delete this
           information.
-        </p>
+        </p><ins cite="#a7">
+        <p>
+          In line with this principle, recipients are strongly encouraged to
+          request the lowest level of location accuracy that is sufficient for
+          their application's functionality. For instance, if an application
+          only needs to know the user's city, it should request an
```

Comment:
> it should request

nit: Avoid "should" since it has special meaning for normative sections (it's good practice even though this section is non-normative). Maybe:

> For instance, if an application only needs to know the user's city, it can respect user privacy by requesting an [=approximate position=].